geopackage_validator/validations/geometry_valid_check.py

```diff
@@ -6,15 +6,16 @@
 )
 
 
-def geometry_valid_check_query(dataset) -> Iterable[Tuple[str, str, str]]:
+def geometry_valid_check_query(dataset) -> Iterable[Tuple[str, str, str, str]]:
     columns = dataset.ExecuteSQL(
         "SELECT table_name, column_name FROM gpkg_geometry_columns;"
     )
     for column in columns:
         validations = dataset.ExecuteSQL(
             'select ST_IsValidReason("{column_name}") as reason, '
             "'{table_name}' as table_name, "
-            "'{column_name}' as column_name "
+            "'{column_name}' as column_name, "
+            "cast(rowid as text) as row_id "
```

Comment:
Waarom cast je de id naar text?

Reply:
rowid is inderdaad een integer. ik heb de cast vervangen door een cast naar INTEGER. De cast is toegevoegd omdat gdal anders de rowid niet retourneert. Waarom niet, heb ik niet kunnen achterhalen